extensions/amp-font/amp-font.md

```diff
@@ -26,9 +26,16 @@ limitations under the License.
 
 # amp-font
 
-## Behavior
+## Do not use
```

Comment:
Replace with a sentence: "This component has been deprecated and should no longer be used. Instead, ..."